.github/workflows/release.yml

```diff
@@ -89,8 +91,8 @@ jobs:
           cd "${GITHUB_WORKSPACE}/src-tauri"
           echo "releaseName=Tari Universe ${{ env.BETA_STRING }} v__VERSION__-${BETA_DATE}" >> $GITHUB_ENV
           # Don't mess with the double quotes and inner escaped quotes
-          yq eval ".productName = \"Tari Universe ${{ env.BETA_STRING }} - ${BETA_DATE} \"" --output-format=json -i tauri.conf.json
-          yq eval ".mainBinaryName = \"Tari Universe ${{ env.BETA_STRING }} - ${BETA_DATE} \"" --output-format=json -i tauri.conf.json
+          yq eval ".productName = \"${{env.BINARY_NAME}} ${{ env.BETA_STRING }}${{env.BETA_DATE_SUFFIX}}${BETA_DATE} \"" --output-format=json -i tauri.conf.json
```

Comment:
Can we add a space around all the envs, ie: ```${{env.BINARY_NAME}}``` to ```${{ env.BINARY_NAME }}```

A little easier to read and for all envs going down